server_tests/test/clientTests.js

```diff
@@ -32,7 +31,8 @@ describe("RUST API TEST SUITE", function() {
     var isBublcam = testModel === camModels.BUBLCAM1_0 || testModel === camModels.BUBLCAM1_2 || testModel === camModels.BUBLMOCK;
     var isMock = testModel === camModels.BUBLMOCK;
     var testViaWifi = process.env.SCARLET_TEST_WIFI === '1';
-    function shouldNotGetHere(err) {
+    var timeoutValue = testViaWifi? 45000 : 30000;
```

Comment:
HDR perhaps... but the original code already accommodates for this by multiplying `timeoutValue` by 2. The only reason we increased it to 45 is to give more leeway.